delta-app/src/main/java/io/cdap/delta/store/DraftService.java

```diff
@@ -229,16 +231,30 @@ public TableAssessmentResponse assessTable(DraftId draftId, Configurer configure
 
     Map<String, TableTransformation> tableLevelTransformations =
       TransformationUtil.getTableLevelTransformations(deltaConfig);
-    List<Transformation> transformations = TransformationUtil.loadTransformations(configurer,
-                                                                                  tableLevelTransformations,
-                                                                                  selectedTable);
+    Map<String, List<Transformation>> transformations = new HashMap<>();
+    List<Problem> transformationErrors = new ArrayList<>();
+    try {
+      transformations = TransformationUtil.loadTransformations(configurer, tableLevelTransformations, selectedTable);
+    } catch (TransformationException e) {
+      transformationErrors.add(new Problem("Transformation Loading failed",
+                               String.format("Failed to load transformations for the table : %s and column : %s " +
+                               "with error : %s", e.getTableName(), e.getColumnName(), e.getMessage()),
+                               "Please ensure the applied transformation's plugin is uploaded'",
+                               ""));
+
+      return new TableAssessmentResponse(Collections.emptyList(), Collections.emptyList(), transformationErrors);
+    }
 
     try (TableRegistry tableRegistry = createTableRegistry(draftId.getNamespace(), deltaConfig, configurer);
          TableAssessor<TableDetail> sourceTableAssessor = createTableAssessor(configurer, deltaConfig.getSource(),
            deltaConfig.getTables());
          TableAssessor<StandardizedTableDetail> targetTableAssessor = createTableAssessor(configurer, target,
            deltaConfig.getTables())) {
-      return assessTable(selectedTable, tableRegistry, sourceTableAssessor, targetTableAssessor, transformations);
+
+      TableAssessmentResponse assessment = assessTable(selectedTable, tableRegistry, sourceTableAssessor,
+                                                       targetTableAssessor, transformations);
+      assessment.getTransformationIssues().addAll(transformationErrors);
```

Comment:
we dont need this now right? since transformationErrors would always be empty here?